pyefis.xml

```diff
@@ -1,147 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
That's correct.  I renamed it and changed it a bit but it's essentially the same file.